openedx/adg/lms/webinars/models.py

```diff
@@ -31,13 +27,18 @@ class WebinarQuerySet(models.QuerySet):
     """
 
     def delete(self):
-        cancelled_upcoming_webinars = self.filter(
-            is_cancelled=False, start_time__gte=now()
-        ).select_related('presenter').prefetch_related('co_hosts', 'panelists')
+        cancelled_upcoming_webinars = self.upcoming_webinars()\
+            .select_related('presenter').prefetch_related('co_hosts', 'panelists')
         send_cancellation_emails_for_given_webinars(cancelled_upcoming_webinars)
         cancel_reminders_for_given_webinars(cancelled_upcoming_webinars)
         self.update(is_cancelled=True)
 
+    def upcoming_webinars(self):
+        return self.filter(is_cancelled=False, end_time__gte=now())
+
+    def delivered_webinars(self):
+        return self.filter(is_cancelled=False, end_time__lt=now())
+
```

Comment:
Since you have added these in the WebinarQueryset class, it is appropriate that you update its docstring as well.

Reply:
@muak50 I think you missed this comment.